navigator_engine/common/graph_loader.py

```diff
@@ -193,11 +195,25 @@ def import_data(sheet_name, graphs):
             model.db.session.add(edge_false)
             model.db.session.commit()
 
+
 def _map_excel_boolean(boolean):
     if boolean in ['TRUE', 1, "1", "T", "t", "true", "True"]:
         return True
-    elif boolean in ['FALSE', 0, "0", "F", "f", "false", "False"] or np.isnan(boolean):
+    elif boolean in ['FALSE', 0, "0", "F", "f", "false", "False", ''] or np.isnan(boolean):
         return False
     else:
         raise ValueError('Value {} read from Initial Graph Config is not valid; Only TRUE and FALSE are valid values.'
                          .format(boolean))
+
+
+def _markdown_to_html(md_in):
+    if type(md_in) != str:
+        if np.isnan(md_in):
+            return None

```

Comment:
I think you can replace this with the more concise:
```suggestion
    if pd.isnull(md_in):
            return None
```